.github/workflows/build-and-test.yml

```diff
@@ -61,3 +61,23 @@ jobs:
             -W clippy::absurd_extreme_comparisons
             -W clippy::erasing_op
             -A clippy::collapsible_else_if
+      - name: prepare AppleSilicon build
+        uses: actions-rs/toolchain@v1
+        with:
+          target: aarch64-apple-darwin
+        if: runner.os == 'macOS'
+
+      # https://github.com/actions/virtual-environments/issues/2557#issuecomment-769611326
+      - if: runner.os == 'macOS'
+        run: |
+          sudo xcode-select -s /Applications/Xcode_12.4.app &&
+          sudo rm -Rf /Library/Developer/CommandLineTools/SDKs/*
+
+      - name: Check compilation for Apple Silicon
+        uses: actions-rs/cargo@v1
+        env:
+          RUSTFLAGS: "-Ctarget-feature=+aes"  # aarch64-apple-darwin does not support ssse3
```

Comment:
It is Ok to remove `RUSTFLAGS` completely because thay have already been defined in `.cargo/config.toml`.

Reply:
Remove the `RUSTFLAGS` in the global scope.

Reply:
Removed `RUSTFLAGS` as suggested.